### DISCUSSION TITLE
git 操作请教

### DISCUSSION THREAD

**Discussion by 37wjy** [BoomingTech/Piccolo#342] [General]
我是一个彩笔 
请问 如果我要fork一个repo到我自己的账号下 
然后从自己账号下clone到本地后做修改 
怎样才能不回推回到booming这边。

有点浪费大家时间了 抱歉

**Comment by Ol6rin** [comment]
可以从GitHub直接Fork，在GitHub Desktop客户端里直接建立新分支开发，新分支创建时会提示你选择是否需要贡献回这里。

**Reply by 37wjy** [reply]
我clone了fork的分支，本地 git remote -v 显示

origin  https://github.com/37wjy/Pilot.git (fetch)
origin  https://github.com/37wjy/Pilot.git (push)
upstream        https://github.com/BoomingTech/Piccolo.git (fetch)
upstream        https://github.com/BoomingTech/Piccolo.git (push)

我从games104/homework02-rendering 创建一个新的分支 homework02-rendering

然后改了些东西后执行 git push --set-upstream origin homework02-rendering ，命令行会提示 已push并创建pull request 请问我要怎么操作 才能让他出现确认创建的选项而不是直接创建

**Reply by Ol6rin** [reply]
我们这边没有收到那个pull request，这个不用太担心。
建立新分支的是否贡献回原始仓库的选项GitHub Desktop客户端界面操作提供额外的选项，git命令行本身不提供。

**Reply by 37wjy** [reply]
好的 谢谢
